src/auro-menuoption.js

```diff
@@ -117,7 +117,7 @@ export class AuroMenuOption extends LitElement {
       ${this.selected && !this.nocheckmark ? html`
         <${this.iconTag}
           ondark
-          customSize
+          customsize
```

Comment:
This attribute needs to be removed, as the `customSize` attribute is no longer supported.